5.sc_Morphology_Logistic_Regression/1.train_models/README.md

```diff
@@ -0,0 +1,23 @@
+# Training models
+
+Each logistic regression model run with parameters from the config file for each type of model:
```

Comment:
I am having trouble understanding the meaning of this sentence. Please consider rewording!